Make.inc

```diff
@@ -871,6 +871,16 @@ else
 ISX86:=0
 endif
 
+
+#If nothing is set default to native
+ifeq ($(MARCH)$(MCPU)$(MTUNE)$(JULIA_CPU_TARGET),)
+MARCH=native
```

Comment:
I think march=native doesn't exist on aarch64?

Reply:
I think this was an M1 issue but I have to check.

Reply:
Also, I don't have a reference at hand on the phone, but on aarch64 you shouldn't set march at all, that's wrong (besides march=native specifically giving an error)

Reply:
https://community.arm.com/arm-community-blogs/b/tools-software-ides-blog/posts/compiler-flags-across-architectures-march-mtune-and-mcpu

With apple clang 14 it built fine but not sure for other versions

Reply:
This is about mtune Vs march in general: https://lemire.me/blog/2018/07/25/it-is-more-complicated-than-i-thought-mtune-march-in-gcc/. But I'm pretty sure there was something about aarch64 specifically. In BinaryBuilder we don't use march for many arm targets https://github.com/JuliaPackaging/BinaryBuilderBase.jl/blob/8ce1e2700dae921aff69cd12b9aef4429913e90c/src/Platforms.jl#L118-L173

Reply:
From ARM itself they say march is fine.
> As long as you’re not cross compiling, the simplest and easiest way to get the best performance on Arm with both GNU compilers and LLVM-compilers is to use only -mcpu=native and actively avoid using -mtune or -march.

> -march=X: Tells the compiler that X is the minimal architecture the binary must run on.  The compiler is free to use architecture-specific instructions.  This flag behaves differently on Arm and x86.  On Arm, -march does not override -mtune, but on x86 -march will override both -mtune and -mcpu.
> -mtune=X: Tells the compiler to optimize for microarchitecture X but does not allow the compiler to change the ABI or make assumptions about available instructions.  This flag has the more-or-less the same meaning on Arm and x86. 
> -mcpu=X: On Arm, this flag is a combination of -march and -mtune.  It simultaneously specifies the target architecture and optimizes for a given microarchitecture.  On x86 this flag is a deprecated synonym for -mtune.